modules/touch/src/win32_touch.cpp

```diff
@@ -119,9 +169,18 @@ Win32TouchHook::Win32TouchHook(void* nativeWindow)
         return;
     }
 
+    //HACK: This hack is required as long as our GLFW version is based on the touch
+    //branch. There is no convenient way to set a GLFWBool (uint32_t) which sets the state
+    //of touch-to-mouseinput interpretation. It happens to be 116 bytes into an internal
+    //glfw struct...
+    uint32_t* HACKY_PTR = (uint32_t *)GetPropW(hWnd, L"GLFW");
```

Comment:
I understand that it is hacky, but maybe rename to something else :)

Reply:
This ptr will *hopefully* only live until the new SGCT stuff is merged. So I'll keep the name as it's a very eye-stabbing name. If anyone stumbles upon it you'll want to remove it! 👍